cpp/deplex/src/deplex/plane_extractor.cpp

```diff
@@ -439,6 +468,45 @@ Eigen::VectorXi PlaneExtractor::Impl::toImageLabels(std::vector<int32_t> const&
   return labels.reshaped<Eigen::RowMajor>();
 }
 
+void PlaneExtractor::Impl::refineLabels(Eigen::MatrixX3f const& pcd_array, Eigen::VectorXi& labels) {
```

Comment:
There are two ways to pass mutable data. One is a non-const reference just as you did, the other one is by passing a pointer. It is arguably which way is better, but in our library we stick to passing a pointer